cypress/e2e/editing/editing.spec.ts

```diff
@@ -55,14 +60,269 @@ describe("Editing", () => {
+        cy.get(".mx_EventTile_edited").should("be.visible");
+
+        cy.get(".mx_RoomView_MessageList").within(() => {
+            // Assert that the message was edited
+            cy.contains(".mx_EventTile", "Massage")
+                .should("exist")
+                .within(() => {
+                    // Click to display the message edit history dialog
+                    cy.contains(".mx_EventTile_edited", "(edited)").click();
+                });
+        });
+
+        cy.get(".mx_Dialog").within(() => {
+            // Assert that the message edit history dialog is rendered
+            cy.get(".mx_MessageEditHistoryDialog").within(() => {
+                // Assert that CSS styles which cannot be detected with snapshots are applied as expected
+                cy.get("li").should("have.css", "clear", "both");
+                cy.get(".mx_EventTile")
+                    .should("have.css", "max-width", "100%")
+                    .should("have.css", "clear", "both")
+                    .should("have.css", "position", "relative")
+                    .should("have.css", "padding-block-start", "0px");
+                cy.get(".mx_EventTile .mx_MessageTimestamp")
+                    .should("have.css", "position", "absolute")
+                    .should("have.css", "inset-inline-start", "0px")
+                    .should("have.css", "text-align", "center");
+                cy.get(".mx_EventTile .mx_EventTile_content").should("have.css", "margin-inline-end", "0px");
+
+                // Assert that the date separator is rendered
+                cy.get("li:nth-child(1) .mx_DateSeparator").within(() => {
+                    cy.get("h2").should("have.text", "Today");
+                });
+
+                // Assert that the edited message is rendered
+                cy.get("li:nth-child(2) .mx_EventTile").within(() => {
+                    cy.get(".mx_EventTile_content").within(() => {
+                        cy.get(".mx_EventTile_body").should("have.text", "Meassage");
+
+                        cy.get(".mx_EventTile_body").within(() => {
+                            // "e" was replaced with "a"
+                            cy.get(".mx_EditHistoryMessage_deletion").should("have.text", "e");
+                            cy.get(".mx_EditHistoryMessage_insertion").should("have.text", "a");
+                        });
+                    });
+                });
+
+                // Assert that the original message is rendered
+                cy.get("li:nth-child(3) .mx_EventTile").within(() => {
+                    cy.get(".mx_EventTile_content .mx_EventTile_body").should("have.text", "Message");
+                });
+            });
+        });
+
+        // Exclude timestamps from a snapshot
+        const percyCSS = ".mx_MessageTimestamp { visibility: hidden !important; }";
+
+        // Take a snapshot
+        cy.get(".mx_Dialog .mx_MessageEditHistoryDialog").percySnapshotElement("Message edit history dialog", {
```

Comment:
I would agree with that checking [these CSS styles](https://github.com/matrix-org/matrix-react-sdk/pull/10406/files#diff-61341016794840db0e9ca539fbd11562023a1da08052fdc8f8781972df746071R99-R102) of `.mx_EventTile` is unnecessary here, as they are not overridden by [_MessageEditHistoryDialog.pcss](https://github.com/matrix-org/matrix-react-sdk/blob/5c47142dab202a48aec6ef1369c7e0ab079107e4/res/css/views/dialogs/_MessageEditHistoryDialog.pcss#L55-L75).

Rather this line should be checked in case of a regression after removing `!important` flag eventually, considering Percy test is not going to be executed on each commit:

https://github.com/matrix-org/matrix-react-sdk/blob/5c47142dab202a48aec6ef1369c7e0ab079107e4/res/css/views/dialogs/_MessageEditHistoryDialog.pcss#L57

Since we do not have a Percy test for timestamps yet (timestamps are hidden to prevent a flaky result), checking their style rules should be worth. I cannot remember exactly when, but I think the position has regressed once due to an issue related to CSS style rules inheritance.

Reply:
This commit https://github.com/matrix-org/matrix-react-sdk/pull/10406/commits/203c8c80a7368623c6e584947f91db7e0d4af538 addresses this.